### DISCUSSION TITLE
Abandon Valve Formatting

### DISCUSSION THREAD

**Discussion by element109** [ReactiveDrop/reactivedrop_public_src#130] [Ideas]
I propose we abandon Valve formatting in favor of the automatic formatting in Visual Studio. It would save numerous unnecessary commits and generally make the project more readable and easier to maintain.

**Comment by element109** [comment]
Or ask valve for their Visual Studio config file.
![Config](https://user-images.githubusercontent.com/20000444/175436223-456e6a15-4631-4d3c-8742-542f890a2420.jpg)

**Reply by mithrand0** [reply]
I am for more modern formatting. 

However, when doing this, I am also for doing a one time reformat of the whole project. Preferably at a moment when there are no open PR's and not much effort in progress.

After this is done, we could hook up i.e. [SonarCloud](https://docs.sonarcloud.io/), that does automated code style checks in the CICD.

**Comment by element109** [comment]
Since I posted this I created a project formatting file see above. However since you guys are back to banning mods again, I am no longer working on this project or even playing the game anymore.

**Reply by mithrand0** [reply]
The majority of users with custom clients sadly choose to use this feature to hack, crash, abuse the game, even on ranked servers. We had many complaints from steam users for this. 

The moment we can differentiate between a cheat client and a normal modded client we are happy to allow them. Right now, we cannot.

That said, happy to discuss this somewhere. As said, we are open for suggestions, as long as cheaters/hackers stay out of the game. Until then, it will be up to the server owner to allow or not.

**Comment by element109** [comment]
Cancelled List
- Port to Source 2013
- New Dedicated Server Browser
- New HUD
- New Spectator Mode
- Particle Crash Fix
- Assorted other bug fixes
- Chat Spam Blocker
- Chat Steam ID Blocker

Traded for a broken anti-cheat.
Smart!
